desktop/libs/indexer/src/indexer/indexers/sql_tests.py

```diff
@@ -579,3 +582,31 @@ def test_generate_create_empty_kudu_table():
   `vrfcn_city_lon` double , PRIMARY KEY (acct_client)
 )   STORED AS kudu TBLPROPERTIES("transactional" = "false")
 ;''' in sql, sql)
+
+
+def test_create_ddl_with_abfs():
+  try:
+    from beeswax.server import dbms
+    from azure.conf import ABFS_CLUSTERS
+  except ImportError as e:
+    raise SkipTest('Hive and HiveServer2 interfaces are not enabled')
+
+  finish = ABFS_CLUSTERS.set_for_testing(
+    {
+      'default': {
+        'fs_defaultfs': 'abfs://my-data@yingstorage.dfs.core.windows.net',
```

Comment:
Nice test!!

Also could add a very similar one with s3 to check that we don't mess-it up?

Reply:
S3 doesn't have same issue here. Only the abfs URL got @yingstorage.dfs.core.windows.net remove from the path, so Impala or Hive throws InvalidUriAuthorityException for abfs://my-data/test_data/cars.csv.

Reply:
I know, the idea was to make sure we don't mess-up the non abfs path..